src/templates/add_chatroom_modal.js

```diff
@@ -32,7 +33,7 @@ export default (o) => html`
                         ${ (o.muc_roomid_policy_error_msg) ? html`<label class="roomid-policy-error">${o.muc_roomid_policy_error_msg}</label>` : '' }
                         <input type="text" required="required" name="chatroom" class="form-control roomjid-input" placeholder="${o.chatroom_placeholder}"/>
                     </div>
-                    ${ o.muc_roomid_policy_hint ?  html`<div class="form-group">{{o.muc_roomid_policy_hint}}</div>` : '' }
```

Comment:
This makes me nervous :)

Please use the XSS library to filter the policy hint for malicious scripts before passing it to `unsafeHTML`.

Reply:
It's already there: https://github.com/conversejs/converse.js/blob/master/src/converse-muc-views.js#L545

Reply:
ah ok, can you please move that to the template? 

I think it makes sense to have it all in one place, then one can be sure that the HTML is being escaped.

Reply:
Done